libcontainer/container_linux.go

```diff
@@ -268,6 +314,34 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 	return cmd, nil
 }
 
+// newCreateProcess is the same as newInitProcess except the INITTYPE value
+// will differ.
+func (c *linuxContainer) newCreateProcess(p *Process, cmd *exec.Cmd, parentPipe, childPipe *os.File) (*initProcess, error) {
+	cmd.Env = append(cmd.Env, "_LIBCONTAINER_INITTYPE="+string(initCreate))
+	nsMaps := make(map[configs.NamespaceType]string)
+	for _, ns := range c.config.Namespaces {
+		if ns.Path != "" {
+			nsMaps[ns.Type] = ns.Path
+		}
+	}
+	_, sharePidns := nsMaps[configs.NEWPID]
+	data, err := c.bootstrapData(c.config.Namespaces.CloneFlags(), nsMaps, "")
+	if err != nil {
+		return nil, err
+	}
+	return &initProcess{
+		cmd:           cmd,
```

Comment:
I see [this line](https://github.com/opencontainers/runc/pull/506/files#diff-2c3b9c3a47f4c72f81e8bdaf26c83d66R146) answering my question partly.